crates/libp2p-networking/tests/counter.rs

```diff
@@ -64,7 +64,7 @@ pub async fn counter_handle_network_event(
     match event {
         IsBootstrapped => {}
         GossipMsg(m, _) | DirectResponse(m, _) => {
-            if let Ok(msg) = bincode_opts().deserialize::<CounterMessage>(&m) {
+            if let Ok(msg) = Serializer::<0, 1>::deserialize::<CounterMessage>(&m) {
```

Comment:
I think this (+ the other `<0,1>` and `STATIC_V_0_1` in this file) are hardcoded to a version, even though the function they're called in are not explicitly versioned.

Reply:
... this one, specifically, might be a problem. I was less stringent about using real versions in tests, but I'm not sure this is uniform in all directions. I'll take a closer look.